src/parseSignedMessage.js

```diff
@@ -23,26 +23,6 @@ function parseSignedMessage (signedMessage) {
   }
 
   switch (msg.verifierFunctionName) {
-    case 'ethToToken':
```

Comment:
I think we should keep these here. Since qualifiers will use this new SDK, they will need the parseSignedMessage to work for older verifiers